### DISCUSSION TITLE
My Favorite Superhero

### DISCUSSION THREAD

**Discussion by Prajwal0225** [Prajwal0225/My-Favorite-SuperHero#43] [Announcements]
🚀 "My Favorite Superhero" is designed with beginners in mind. It's a web project built using HTML, CSS, JavaScript, and the Superhero API. Whether you're new to web development or open source, this project is perfect for you.

**Comment by 22Yash** [comment]
Hey .
I am starting open source contribution just  can you see me pull request and suggest any improvement.
